packages/docusaurus-theme-openapi-docs/src/theme/SchemaTabs/styles.module.css

```diff
@@ -75,7 +77,7 @@
   background-color: var(--ifm-color-info);
 }
 
-.schemaTabActive {
+.tabItem.schemaTabActive {
```

Comment:
@sserrata  I might've jumped the gun on suggesting to remove `border` and `color` from `.tabItem` (that also removed the normal state tab styling 😅 ). I've increased the specifier to correct the active tab styling.

Reply:
No worries, thanks for the improvement!